app/search_builders/hyrax/single_result.rb

```diff
@@ -4,6 +4,7 @@ module SingleResult
 
     included do
       self.default_processor_chain += [:find_one]
+      self.default_processor_chain.delete :filter_models
```

Comment:
I wonder if this would be any clearer if we did:
```
self.default_processor_chain = default_processor_chain.dup + [:find_one] - [:filter_models]
```
thoughts?

Reply:
I'm wondering also if this exposes odd models that we wouldn't otherwise want to show? (e.g. AdminSets, Access controls, etc.)  Might it be better if you just overrode `filter_models` to accept everything in your own application?

Reply:
I personally don't find that clearer, but that's just me. I kinda like one line per mutation. Also, I don't see why re-assigning `self.default_processor_chain` to a dup of itself is necessary. Is it?

Reply:
Actually, I think `self.default_processor_chain -= [:filter_models]` might be clearer, or at least more consistent with how the default processor chain is typically changed in methods like these.

Reply:
I'm cool with `-=` too 👍